normandy/selfrepair/static/js/JexlEnv.js

```diff
@@ -0,0 +1,16 @@
+import { Jexl } from 'jexl';
+import { stableSample } from './utils.js';
+
+export default class JexlEnv {
```

Comment:
I kind prefer using the full word (Environment in this case) for classnames and abbreviate for variables when convenient. This is kind of a personal nit though.